### PR TITLE
Feature: Implement new design for `<BusinessList />` page (HAIR-100)

### PR DESCRIPTION
**Pull request by CodeMeKathy**
## Description
This Pull Request (PR) features new styles within the `<BusinessList />` component page that match the new design spec.

## Acceptance Criteria
- [x] Update the UI to reflect the redesign for the `<BusinessList />` component page to match the new design spec.

## Related Issue/Ticket

Closes #100

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|   ✅   | :bug: Bug fix(es)              |
| ✅    | :sparkles: New feature     |
| ✅    | :hammer: Refactoring       |
|     | :100: Add tests            |
|  ✅  | :link: Update dependencies |
|     | :scroll: Docs              |

## UI Updates
<details>
<summary>Revised Design Mockup</summary>

- ## Current Business List Design
searchresultspage-updated

</details>

<details>

<summary>PR Revisied Business List Page</summary>

- ## Updated Business List Page UI

Updated Business List Page UI

</details>

<details>
<summary>Previous Design Mockup</summary>

- ## Previous Business List Design 

searchresultspage
</details>

<details>

<summary>Previous Business List Page Update</summary>

Previous Business List Page Updat

</details>

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -8,26 +14,46 @@ body {
...
+  display: flex;
   height: 100vh;
   width: 100%;
-  max-width: 1206;
-  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+
+  /* TODO-FIXME: Removed because it prevented children components' page scroll*/
```
Comment (cynthiaeddy) [inline client/src/App.css:38]: please add back in position: fixed. each page is fixed, while children components have their own nested scrolls as per figma docs. 
max-width: 1206 please remove. (mistake on my part)

File: client/src/App.css [hunk 2]
```diff
@@ -8,26 +14,46 @@ body {
...
-  height: 100px;
-  background: none;
-  top: 0;
-  z-index: 10;
-  border-bottom: 0.5px solid black;
   justify-content: space-around;
+  border-bottom: 0.5px solid black;
+  background-color: inherit;
+  /* background: url(./assets/images/pinkcrop1.jpg); */
+  /** Removed to allow for overall repositioning of the parent container layout with the navbar and other internal sibling components, i.e. <BusinessList/> or <AddBusiness/> */
```
Comment (cynthiaeddy) [inline client/src/App.css:51]: is this relevant anymore as parent container is fixed? please explain if so, thanks!

File: client/src/pages/BusinessList/BusinessList.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
+.BusinessList-Wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  margin-top: -10vh; /* Adds overlap styles to navbar */
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  background-position: bottom center;
+  background-image: url(../../assets//images/whitecrop\ 1.png);
```
Comment (cynthiaeddy) [inline client/src/pages/BusinessList/BusinessList.css:10]: please use the hairwhite.jpg image, and follow the .home container (on home page) as far as your businessList-Wrapper specs. 
You need to incorporate the css grid and as per the figma docs (see attached screenshot).  Can do a zoom if you want.
no rounded corners on boxes, no shadows. 
Screen Shot 2022-03-24 at 3 40 10 PM

File: client/src/components/BusinessDetails/Review/ReviewElements.tsx [hunk 4]
```diff
@@ -10,6 +10,8 @@ export const Container = styled.div`
 export const StarsContainer = styled.div`
   margin-top: 1rem;
   display: flex;
+  justify-content: center;
+  align-items: center;
   font-size: 1.5rem;
```
Comment (cynthiaeddy) [inline client/src/components/BusinessDetails/Review/ReviewElements.tsx:15]: please adjust star size as per figma doc.

File: client/src/components/FilterServicesAndFeatures/FilterServicesAndFeatures.tsx [hunk 5]
```diff
@@ -49,64 +53,52 @@ const FilterServicesAndFeatures: React.FC<Props> = (props: Props) => {
...
-                    name={`feature-${feature[0]}`}
-                    id={feature[1]}
-                    defaultChecked={feature[2].isChecked}
-                    value={id}
-                    onChange={onFeatChange}
-                  />
-                  <label className='form-check-label' htmlFor={feature[1]}>
+                <div className='Filters-FormCheck' key={`${feature}_` + index}>
+                  <input className='Filters-FormCheckInput' type='checkbox' name={`feature-${feature[0]}`} id={feature[1]} defaultChecked={feature[2].isChecked} value={id} onChange={onFeatChange} />
+                  <label className='Filters-FormCheckLabel' htmlFor={feature[1]}>
```
Comment (cynthiaeddy) [inline client/src/components/FilterServicesAndFeatures/FilterServicesAndFeatures.tsx:65]: This looks great overall. I know figma docs don't show the reset filter button, so will style that later, as well as adding text alignment.

File: client/src/pages/Home/Home.css [hunk 6]
```diff
@@ -8,16 +8,38 @@
   grid-column-gap: 20px;
```
Comment (cynthiaeddy) [inline client/src/pages/Home/Home.css:8]: this is all updated in my new PR.

**Review by cynthiaeddy — CHANGES_REQUESTED** [review]
Wow you did a lot! Please review my PR, merge it with your PR to incorporate the updated styles. I think (hope!) using the grid will make styling much easier.

File: client/src/components/FilterServicesAndFeatures/FilterServicesAndFeatures.tsx (hunk 5 above)
Comment (CodeMeKathy) [reply client/src/components/FilterServicesAndFeatures/FilterServicesAndFeatures.tsx:65]: Thank you @cynthiaeddy! I have updated the layout to include the button styles.

File: client/src/pages/BusinessList/BusinessList.css (hunk 3 above)
Comment (CodeMeKathy) [reply client/src/pages/BusinessList/BusinessList.css:12]: Hi @cynthiaeddy! I do not see a `hairwhite.jpg image` in the figma files. The image named in the provided picture and the one I used is `whitecrop 1`. If there is a different one that should be used please add it to the figma and I will update it. Thank you!

**Comment by cynthiaeddy** [comment]
Works well, thanks!

**Review by cynthiaeddy — APPROVED** [review]

**Review by sergiobtos — APPROVED** [review]
As this branch is about design, i will go with Cynthia.

**Pull request merged**